testing/renovation-npm/react-app/src/App.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-unused-vars */
-import 'devextreme/dist/css/dx.light.css';
+import '@devextreme/react/scss/bundles/dx.light.scss';
```

Comment:
I don't think we need `bundle` folder here. Lets discuss with PO and techwriter.

Reply:
done